Description: We reverted the disabling, so we can also the disable the
 re-enabling. We can't remove it because the re-enabling does not re-enable
 everything.
 .
 This reverts commit b264c77ae98247adf4f99f75a37393232a55e580.
Author: Pascal Roeleven <dev@pascalroeleven.nl>

--- a/debian/changelog
+++ b/debian/changelog
@@ -52,7 +52,6 @@
   * Use CFLAGS_NODIST, LDFLAGS_NODIST for passing build flags.
   * d/rules: Move configure/build targets for each build together.
   * Drop ffi configure options, obsolete.
-  * Re-enable PGO/LTO builds.
 
  -- Matthias Klose <doko@debian.org>  Tue, 02 Apr 2024 16:53:02 +0200
 
--- a/debian/rules
+++ b/debian/rules
@@ -231,6 +231,8 @@
     with_pgo =
     with_lto =
 endif
+with_pgo =
+with_lto =
 
 BASE_LTO_CFLAGS = -g -fwrapv -O3
 ifeq ($(with_lto),yes)
@@ -253,6 +255,7 @@
 	-e "s/-specs=[^ '\"]*//g" \
 	-e "s/-fdebug-prefix-map=[^ '\"]*//g" \
 	-e "s/-ffile-prefix-map=[^ '\"]*//g" \
+	-e 's/-Wl,-z,relro//g' \
 	-e 's,^RUNSHARED *=.*,RUNSHARED=,' \
 	-e '/BLDLIBRARY/s/-L\. //'
 
